examples/basic-examples/.eslintrc.js

```diff
@@ -28,6 +28,10 @@ module.exports = {
     // allow async-await
     'generator-star-spacing': 'off',
     // allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'quotes': ['off', 'single', { 'allowTemplateLiterals': true }],
+    'quote-props': 'off',
+    'dot-notation': 'off',
+    'prefer-const': 'off'
```

Comment:
@rubyllamadora I think we will need these `.eslintrc` update in the update script.